ios/FluentUI.Demo/FluentUI.Demo/DemoColorThemeWindows.swift

```diff
@@ -6,70 +6,74 @@
 import FluentUI
 import UIKit
 
-class DemoColorThemeDefaultWindow: UIWindow, ColorProviding {
-    func primaryColor(for window: UIWindow) -> UIColor? {
-        return Colors.communicationBlue
-    }
-
-    func primaryTint10Color(for window: UIWindow) -> UIColor? {
-        Colors.Palette.communicationBlueTint10.color
-    }
-
-    func primaryTint20Color(for window: UIWindow) -> UIColor? {
-        Colors.Palette.communicationBlueTint20.color
-    }
-
-    func primaryTint30Color(for window: UIWindow) -> UIColor? {
-        Colors.Palette.communicationBlueTint30.color
-    }
+class DemoColorThemeWindow: UIWindow, ColorProviding {
+    public static var theme: NSString = "Default"
 
-    func primaryTint40Color(for window: UIWindow) -> UIColor? {
-        Colors.Palette.communicationBlueTint40.color
-    }
-
-    func primaryShade10Color(for window: UIWindow) -> UIColor? {
-        Colors.Palette.communicationBlueShade10.color
+    func setTheme(theme: NSString) {
+        DemoColorThemeWindow.theme = theme
     }
 
-    func primaryShade20Color(for window: UIWindow) -> UIColor? {
-        Colors.Palette.communicationBlueShade20.color
-    }
-
-    func primaryShade30Color(for window: UIWindow) -> UIColor? {
-        Colors.Palette.communicationBlueShade30.color
-    }
-}
-
-class DemoColorThemeGreenWindow: UIWindow, ColorProviding {
     func primaryColor(for window: UIWindow) -> UIColor? {
-        return UIColor(named: "Colors/DemoPrimaryColor")
+        if DemoColorThemeWindow.theme == "Green" {
```

Comment:
This is a typical pattern of 2 classes with different implementations of the same interface. We shouldn't break this pattern.